MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -55,3 +55,29 @@ open class NavigationRouteOptions: RouteOptions {
         super.init(coder: decoder)
     }
 }
+
+@objc(MBNavigationMatchOptions)
+open class NavigationMatchOptions: MatchingOptions {
+    
+    @objc public required init(waypoints: [Waypoint], profileIdentifier: MBDirectionsProfileIdentifier? = .automobileAvoidingTraffic) {
+        super.init(waypoints: waypoints.map {
+            $0.coordinateAccuracy = -1
```

Comment:
Can we turn this magic number into a constant? perhaps it might live in an extension? `CLLocationAccuracy.invalid` (or just `.invalid` in this line) would read a lot cleaner than `-1`.